html/parking/back-on-network.html

```diff
@@ -1,4 +1,52 @@
-<h1>Restrictions have been removed.</h1>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="/common/styles.css" media="screen" rel="stylesheet" type="text/css" />
+    <title>Restrictions have been removed</title>
+    <script>
+      function release(target){
+        document.getElementById(target).innerHTML = "One moment please";
+      }
+    </script>
+  </head>
+  <body>
 
-<a href="http://www.packetfence.org">Please click here to access the captive portal</a>
+    <div class="layout layout--center u-pb u-palm-p">
+      <div class="layout__item u-1/2 u-2/3-lap u-1/1-palm">
+        <div class="layout--center">
+          <div class="box box--hero u-m u-mb--">
+            <img class="iconic" src="/common/img/ic_done_24px.svg" alt="Info"
+                 onerror="this.onerror=null; this.src='/common/img/ic_done_72px.png'">
+          </div>
+          <h4>Restrictions have been removed</h4>
+        </div>
 
+        <div class="layout layout--center">
+          <div class="layout__item u-2/3 u-1/1-palm">
+            <a href="/cgi-bin/release.pl" onclick="release('btn-label')" class="btn btn--full btn--light u-mt">
```

Comment:
the href here should point to an external website so the redirection process brings the user back to the normal portal